tests/yast2_gui/yast2_firewall_settings.pm

```diff
@@ -0,0 +1,124 @@
+# SUSE's openQA tests
```

Comment:
for this test we need to break it down in different parts, avoid validation on the UI, and perform those validation with command line tools.At the moment it is impossible to know what this does without looking at the screenshots. The goal is to have something simple like when start/stop the service, we can open and close the module as needed.

Reply:
I was checking and the original test might not make sense for this part, because it is changing the interface and default zone to trusted but then right afterwards is checking in the UI in which zone is the interface, but yast module do not work like that, do not refresh with previous action without re-opening the module.
Therefore changing the interface to trusted with libyui and verify with command line could be a test module, we exclude going to interface and change the zone (because it is probably messing up with this, you need to confirm manually).
The other test module is for adding service and ports to a particular zone and its cmd verification.
Both modules should be fine for tw and sle.